rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
+    hamm_dist = 0
+    for i in range(len(a)):
+        if a[i] != b[i]:
+            hamm_dist += 1
+        else:
+            pass
```

Comment:
`else: pass` here isn't going to do anything different than just leaving it empty. It will reach the end of the `if` statement, do nothing, then continue through the for loop.